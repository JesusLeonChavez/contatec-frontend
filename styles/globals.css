:root{
  --primary: #706F6F;
  --secondary: #304659;
}

html,
body {
  padding: 0;
  margin: 0;

}

a {
  color: inherit;
  text-decoration: none;
}

* {
  box-sizing: border-box;
  font-family: 'Poppins', sans-serif;
}

.generalWrapper{
  max-width: 1300px;
  margin: 0 auto;
  overflow: auto;
  padding: 0 40px;
}

.flexCenterCustom{
  display: flex;
  align-items: center;
  justify-content: center;
}


/*----------------Swiper-----------------------*/
.swiper-container {
  width: 100%;
  height: 100%;
}

.swiper-slide {
  text-align: center;
  font-size: 18px;
  background: red;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}

.slide-2{
  background: green;
}

.mySwiper  .swiper-button-next, .mySwiper  .swiper-button-prev {
  color: white;
  border: 1px solid white;
  border-radius: 50%;
  padding: 1.3rem;
}

.mySwiper .swiper-button-next{
  top: 800px;
  right: 100px;
}

.mySwiper .swiper-button-prev{
  top: 800px;
  right: 180px;
  left: auto;
}

.mySwiper .swiper-button-next::after,.mySwiper .swiper-button-prev::after{
  font-size: 1rem;
}


.mySwiper2{
  width: 200px;
}
.mySwiper2 .swiper-container{
  width: 100%;
  height: 100%;
}
.mySwiper2 .swiper-slide{
  text-align: center;
  font-size: 18px;
  background: #fff;
  background: pink;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}
.mySwiper2 .slide-2{
  background: blue;
}


.mySwiper3{
  width:100%;
}
.mySwiper3 .swiper-container{
  width: 300px;
  height: 100%;
}
.mySwiper3 .swiper-slide{
  text-align: center;
  font-size: 18px;
  background: #fff;
  background: purple;
  height: 500px;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}
.mySwiper3 .slide-2{
  background: blue;
}

.mySwiper3 .swiper-button-next, .mySwiper3 .swiper-button-prev {
  color: white;
  border: 1px solid white;
  border-radius: 50%;
  padding: 1.3rem;
}

.mySwiper3 .swiper-button-next{
  top: 450px;
  right: 100px;
}
.mySwiper3 .swiper-button-prev{
  top: 450px;
  right: 180px;
  left: auto;
}

.mySwiper3 .swiper-button-next::after, .mySwiper3 .swiper-button-prev::after{
  font-size: 1rem;
}




