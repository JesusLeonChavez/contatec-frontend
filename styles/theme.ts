import { extendTheme } from '@chakra-ui/react'
import { ButtonStyles as Button } from './components/buttonStyles'
export const newTheme = extendTheme({
  colors: {
    primary: '#EB0029',
    secondary: '#FF6F91',
    highlight: '#00C9A7'
  },
  components: {
    Button
  }
})
