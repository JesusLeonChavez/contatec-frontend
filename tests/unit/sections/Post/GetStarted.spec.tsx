/**

 * @jest-environment jsdom

 */

import { render } from "@testing-library/react"
import "@testing-library/jest-dom"
import { newTheme } from "../../../../styles/theme"
import { ChakraProvider } from "@chakra-ui/react"
import GetStarted from "../../../../sections/Post/GetStarted"

const ChakraRenderer = ({ children }) => {
  return <ChakraProvider theme={newTheme}>{children}</ChakraProvider>
}

describe("GetStarted", function () {
  test("debe renderizarse el componente", () => {
    const component = render(<GetStarted />, {
      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
      // @ts-ignore
      wrapper: ChakraRenderer
    })

    expect(component).toMatchSnapshot()
  })
})
