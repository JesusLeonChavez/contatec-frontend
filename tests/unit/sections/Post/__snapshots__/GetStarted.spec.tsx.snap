// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetStarted debe renderizarse el componente 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    class="chakra-ui-light"
  >
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <div
        class="infoContainer"
      >
        <div
          class="infoWrapper"
        >
          <div
            class="infoRowNoInvert"
          >
            <div
              class="css-1jhtigs"
            >
              <div
                class="css-1qp29ss"
              >
                <p
                  class="chakra-text bold600 css-1nypnod"
                >
                  ¡Empieza a publicar hoy!
                </p>
                <p
                  class="chakra-text css-mcpu91"
                >
                  Son mas de 2000 personas registradas que como tú ya han publicado su servicio y se encuentran laborando. ¡Qué esperas para unirte y publicar hoy tu primer anuncio!
                </p>
                <a
                  href="/publicar/nuevo_post"
                >
                  <button
                    class="chakra-button css-x6w309"
                    type="button"
                  >
                    Publicar
                  </button>
                </a>
              </div>
            </div>
            <div
              class="css-1i40juj"
            >
              <div
                style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
              >
                <div
                  style="box-sizing: border-box; display: block; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    role="presentation"
                    src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                    style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                  />
                </div>
                <noscript />
                <img
                  alt="OurCategories"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="infoContainer"
    >
      <div
        class="infoWrapper"
      >
        <div
          class="infoRowNoInvert"
        >
          <div
            class="css-1jhtigs"
          >
            <div
              class="css-1qp29ss"
            >
              <p
                class="chakra-text bold600 css-1nypnod"
              >
                ¡Empieza a publicar hoy!
              </p>
              <p
                class="chakra-text css-mcpu91"
              >
                Son mas de 2000 personas registradas que como tú ya han publicado su servicio y se encuentran laborando. ¡Qué esperas para unirte y publicar hoy tu primer anuncio!
              </p>
              <a
                href="/publicar/nuevo_post"
              >
                <button
                  class="chakra-button css-x6w309"
                  type="button"
                >
                  Publicar
                </button>
              </a>
            </div>
          </div>
          <div
            class="css-1i40juj"
          >
            <div
              style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
            >
              <div
                style="box-sizing: border-box; display: block; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  role="presentation"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                  style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                />
              </div>
              <noscript />
              <img
                alt="OurCategories"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`JoinUs debe renderizarse el componente 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    class="chakra-ui-light"
  >
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <div
        class="infoContainer"
      >
        <div
          class="infoWrapper"
        >
          <div
            class="infoRowNoInvert"
          >
            <div
              class="css-1jhtigs"
            >
              <div
                class="css-1qp29ss"
              >
                <p
                  class="chakra-text bold600 css-1nypnod"
                >
                  ¡Empieza a publicar hoy!
                </p>
                <p
                  class="chakra-text css-mcpu91"
                >
                  Son mas de 2000 personas registradas que como tú ya han publicado su servicio y se encuentran laborando. ¡Qué esperas para unirte y publicar hoy tu primer anuncio!
                </p>
                <a
                  href="/publicar/nuevo_post"
                >
                  <button
                    class="chakra-button css-x6w309"
                    type="button"
                  >
                    Publicar
                  </button>
                </a>
              </div>
            </div>
            <div
              class="css-1i40juj"
            >
              <div
                style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
              >
                <div
                  style="box-sizing: border-box; display: block; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    role="presentation"
                    src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                    style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                  />
                </div>
                <noscript />
                <img
                  alt="OurCategories"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="infoContainer"
    >
      <div
        class="infoWrapper"
      >
        <div
          class="infoRowNoInvert"
        >
          <div
            class="css-1jhtigs"
          >
            <div
              class="css-1qp29ss"
            >
              <p
                class="chakra-text bold600 css-1nypnod"
              >
                ¡Empieza a publicar hoy!
              </p>
              <p
                class="chakra-text css-mcpu91"
              >
                Son mas de 2000 personas registradas que como tú ya han publicado su servicio y se encuentran laborando. ¡Qué esperas para unirte y publicar hoy tu primer anuncio!
              </p>
              <a
                href="/publicar/nuevo_post"
              >
                <button
                  class="chakra-button css-x6w309"
                  type="button"
                >
                  Publicar
                </button>
              </a>
            </div>
          </div>
          <div
            class="css-1i40juj"
          >
            <div
              style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
            >
              <div
                style="box-sizing: border-box; display: block; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  role="presentation"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                  style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                />
              </div>
              <noscript />
              <img
                alt="OurCategories"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
